src/extension/src/azure/azure-app-service/appServiceProvider.ts

```diff
@@ -92,6 +143,70 @@ export class AppServiceProvider {
       });
   }
 
+  private getAppServiceARMTemplate(): any {
+    const templatePath = path.join(
+      appRoot.toString(),
+      "src",
+      "azure",
+      "azure-app-service",
+      "arm-templates",
+      "template.json"
+    );
+    return JSON.parse(fs.readFileSync(templatePath, "utf8"));
+  }
+
+  private getAppServiceARMParameter(selections: AppServiceSelections): any {
+    const parameterPath = path.join(
+      appRoot.toString(),
+      "src",
+      "azure",
+      "azure-app-service",
+      "arm-templates",
+      "parameters.json"
+    );
+
+    let parameters = JSON.parse(fs.readFileSync(parameterPath, "utf8"));
+
+    parameters.parameters = {
+      name: {
+        value: selections.siteName
+      },
+      appServicePlanName: {
+        value: selections.appServicePlanName
+      },
+      sku: {
```

Comment:
Just out of curiosity, what is `sku`? or could you direct me to the documentation of the parameters?

Reply:
It is the pricing tier (https://docs.microsoft.com/en-us/azure/search/search-sku-tier)